config/rbac/namespaced/role.yaml

```diff
@@ -28,217 +28,16 @@ rules:
   - atlas.mongodb.com
   resources:
   - atlasbackupcompliancepolicies
-  verbs:
```

Comment:
I guess same question as clusterwide, these removals are surprising to me.

Reply:
Similar as above, these permissions are not removed but due to a smarter controller-runtime binary consolidated as one block: https://github.com/mongodb/mongodb-atlas-kubernetes/blob/e450791421bc0ed866b757deff00fca72fe3ac42/config/rbac/namespaced/role.yaml#L27-L49

tldr: it looks much cleaner now